https://github.com/pytorch/kineto/commit/eb34f147f2af821da931c18457c26b076c8491dd
https://github.com/pytorch/kineto/pull/959
https://github.com/pytorch/kineto/issues/958

From eb34f147f2af821da931c18457c26b076c8491dd Mon Sep 17 00:00:00 2001
From: Aaron Gokaslan <aaronGokaslan@gmail.com>
Date: Fri, 12 Jul 2024 09:23:33 -0700
Subject: [PATCH] Update libfmt to 11-0-0. Closes #958. (#959)

Summary:
Fixes https://github.com/pytorch/kineto/issues/958

Pull Request resolved: https://github.com/pytorch/kineto/pull/959

Reviewed By: davidberard98, yoyoyocmu

Differential Revision: D59637583

Pulled By: aaronenyeshi

fbshipit-source-id: 48def339ed3097ecb4746d7870203040ac535858
--- a/src/Config.cpp
+++ b/src/Config.cpp
@@ -12,6 +12,7 @@
 
 #include <fmt/chrono.h>
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <chrono>
 #include <fstream>
 #include <functional>
--- a/src/CuptiRangeProfilerConfig.cpp
+++ b/src/CuptiRangeProfilerConfig.cpp
@@ -12,6 +12,7 @@
 #include <stdlib.h>
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <ostream>
 
 
--- a/src/DeviceProperties.cpp
+++ b/src/DeviceProperties.cpp
@@ -9,6 +9,7 @@
 #include "DeviceProperties.h"
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <vector>
 
 #if defined(HAS_CUPTI)
--- a/src/EventProfiler.cpp
+++ b/src/EventProfiler.cpp
@@ -10,6 +10,7 @@
 
 #include <assert.h>
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <time.h>
 #include <algorithm>
 #include <cmath>
--- a/test/CuptiActivityProfilerTest.cpp
+++ b/test/CuptiActivityProfilerTest.cpp
@@ -7,6 +7,7 @@
  */
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <folly/json/json.h>
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
--- a/test/CuptiRangeProfilerConfigTest.cpp
+++ b/test/CuptiRangeProfilerConfigTest.cpp
@@ -10,6 +10,7 @@
 #include "src/CuptiRangeProfilerConfig.h"
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <gtest/gtest.h>
 #include <time.h>
 #include <chrono>
--- a/test/CuptiRangeProfilerTest.cpp
+++ b/test/CuptiRangeProfilerTest.cpp
@@ -16,6 +16,9 @@
 #include <fcntl.h>
 #endif
 
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+
 #include "include/libkineto.h"
